### PR TITLE
Add trial project to review

### PR DESCRIPTION
**Pull request by omarsalem7**
Main features of the trial:
- Navbar
- Welcome section
- Project section
- Footer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# my-trial-review
\ No newline at end of file
+# this is my-trial-review
```
Comment (omarsalem7) [inline README.md:1]: Hi 😃
good work but there are little required changes that should be completed:

1- Add a descriptive README file to your project - please use [this template ](https://github.com/microverseinc/readme-template) but remember to customize this template to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <p><button type="button">Add to Cart</button></p>
+      </div>
+      <div class="card">
+        <img src="./low-angle-photography-of-red-concrete-house-3013120.jpg" alt="Denim Jeans">
+        <h1>Tailored Jeans</h1>
+        <p class="price">$19.99</p>
+        <p>Some text about the jeans. Super slim and comfy lorem ipsum lorem jeansum. Lorem jeamsun denim lorem jeansum.
+        </p>
+        <p><button type="button">Add to Cart</button></p>
+      </div>
```
Comment (omarsalem7) [inline index.html:55]: **Optional:**
you can use [BEM CSS ](https://css-tricks.com/bem-101/)to write your code, it will improve the way of writing code.

**Review by omarsalem7 — COMMENTED** [review]
# Changes required ♻️ 
Please check my comments below and make all required changes.

And submit for a re-review when you are done.

Thanks 😃

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
## Hi @omarsalem7   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**